src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int INDEX_FOR_ARRAY_ZERO = 0;
```

Comment:
```suggestion
    private static final int DATE_INDEX = 0;
```
The goal of creating these constants is to make code more readable. 
Is `arrayOfData[INDEX_FOR_ARRAY_ZERO]` better then `arrayOfData[0]`. Is it easier to understand? I do not think so.
But if you have `arrayOfData[DATE_INDEX]` or `arrayOfData[NAME_INDEX]` - anyone understand what is it right away.